spec/system/reward_spec.rb

```diff
@@ -42,4 +42,15 @@
     expect(page).to have_content 'Reward bought.'
     expect(page).to have_content '0'
   end
+
+  it 'pagination check' do
+    create_list(:reward, 18)
+    visit rewards_path
+
+    expect(page).not_to have_content Reward.last.title
+    expect(page).not_to have_content Reward.last.price
```

Comment:
I would check here, if we have 10 rewards on the page

Reply:
true, fixed